src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2237,6 +2237,49 @@
      * The {@code default} keyword.  This element has no
      * children.
      *
+     * <p>For example:</p>
+     * <pre>
+     * switch (3){
+     *   case 1 :
+     *     x=1;
+     *     break;
+     *   default:
+     *     x=3;
+     * }
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--LITERAL_SWITCH -&gt; switch
```

Comment:
you can omit `|--` and start with `LITERAL_SWITCH`

so you are AST tree will look like:

```
     * LITERAL_SWITCH -&gt; switch
     *  |--LPAREN -&gt; (
     * ...and so on
```

Reply:
Also do not forget update your ast tree with update example

Reply:
done 
@timurt